src/types.ts

```diff
@@ -461,3 +461,12 @@ export interface CalculateMetaTransactionQuoteParams {
     slippagePercentage?: number;
     excludedSources?: ERC20BridgeSource[];
 }
+
+export enum TransactionStates {
+    Unsubmitted = 'unsubmitted',
+    Submitted = 'submitted',
+    Mempool = 'mempool',
+    Stuck = 'stuck',
+    Confirmed = 'confirmed',
+    Dropped = 'dropped',
```

Comment:
We're missing one state: `aborted`. When a txn with a specific sender & nonce get's mined, we should check for other txns with those same sender & nonce, and mark them as `aborted`.

Reply:
I have added a routine to mark transactions as `aborted`. Do we want to preserve transactions that have the `dropped` status? Or change their status to `aborted` as well?

Reply:
Change to `aborted`. Elaborated above.

Reply:
@opaolini mind adding a comment next to each other these explaining them? Some might not be  as self-explanatory.

Reply:
👍